### DISCUSSION TITLE
Recommendation: Roadmap 🚀

### DISCUSSION THREAD

**Discussion by hasansezertasan** [jowilf/starlette-admin#341] [Ideas]
Hi everyone 👋 

I'm currently exploring on Starlette Admin, which burns... I'm loving everything about it, reading docs, exploring the source code (mostly sqla section) and trying to contribute. It's my new go-to 👍  for administrative interfaces.

@jowilf did a lot of work here, congratulations 👏 👏 👏 

Title tells the story but let me tell/ask you these:

- What is the end game here? I can see that you said `The main goal of this project is to provide similar tool for Starlette/FastApi.` in the `Alternatives, Inspiration and Comparisons` section. It's enough but still, tell us more. It has features that Flask Admin doesn't have and missing features that Flask Admin have.
- What should we expect for new versions, do you have a sketched plan? If the answer is yes, can you consider sharing it 👉👈? If no, I believe a Roadmap is a good idea. You can create a [Github Project](https://github.com/jowilf/starlette-admin/projects), which could be considered as a roadmap/plan.

I believe this project has huge potential and could be the python standard for administrative interfaces. I don't know if anyone checked it but this project is installed 119,005 times in last month according to [PyPI Stats](https://pypistats.org/packages/starlette-admin) which is more then any other FastAPI/Starlette compatible administrative interfaces.

I really adopted/embraced this project and I feel/want to contribute as much as I can, having a Roadmap/Backlog would probably motivate you and other contributors like me.

**Comment by agxc** [comment]
Thank you @hasansezertasan  for the great summary. I think the project has huge potential too. I want to see this grow and flourish and I intend to use it for a couple of small projects. Like you, I am keen to learn more about what the roadmap looks like.

**Reply by hasansezertasan** [reply]
You're welcome. I totally agree! 🙏
